### PR TITLE
(DOCSP-27121): Release Notes for 12/23/2022

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-27121

### Staged Changes

N/A

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [N/A] Create Jira ticket for corresponding docs-app-services update(s), if any
- [N/A] Checked/updated Admin API
- [N/A] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: docs-release-notes.md [hunk 1]
```diff
@@ -1,13 +1,59 @@
+# December 23, 2022
+
+## Flutter SDK:
+
+- Add App Services tab to the Flutter landing page
+- Add "User Metadata" page to Flutter
+- Remove download progress notification from Flutter docs
+- Add info about connecting Using Android 7 or Older
+- Added info clarifying "completed subscription state
```
Comment (cbullinger) [inline docs-release-notes.md:9]:
```suggestion
- Added info clarifying "completed" subscription state
```

File: docs-release-notes.md [hunk 2]
```diff
@@ -1,13 +1,59 @@
...
+- Add "Manage Email/Password Users" page
+
+## .NET SDK:
+
+- Change capitalization of "Linq" to "LINQ" in QuickStart
+- Added info clarifying "completed subscription state
+
+## Kotlin SDK:
+
+- Added info clarifying "completed subscription state
```
Comment (cbullinger) [inline docs-release-notes.md:24]:
```suggestion
- Added info clarifying "completed" subscription state
```

File: docs-release-notes.md [hunk 3]
```diff
@@ -1,13 +1,59 @@
...
+
+## React Native SDK:
+
+- Added info clarifying "completed subscription state
```
Comment (cbullinger) [inline docs-release-notes.md:28]:
```suggestion
- Added info clarifying "completed" subscription state
```

File: docs-release-notes.md [hunk 4]
```diff
@@ -1,13 +1,59 @@
...
+
+## Swift SDK :
+
+- Added info clarifying "completed subscription state
```
Comment (cbullinger) [inline docs-release-notes.md:32]:
```suggestion
- Added info clarifying "completed" subscription state
```

File: docs-release-notes.md [hunk 5]
```diff
@@ -1,13 +1,59 @@
...
+
+## C++ SDK:
+
+- Add information about building an app with Android
+- Add "Manage Email/Password Users" page
+
+## .NET SDK:
+
+- Change capitalization of "Linq" to "LINQ" in QuickStart
+- Added info clarifying "completed subscription state
```
Comment (cbullinger) [inline docs-release-notes.md:20]:
```suggestion
- Added info clarifying "completed" subscription state
```

Comment (cbullinger) [inline docs-release-notes.md:20]:
```suggestion
- Added info clarifying "completed" subscription state
```

File: docs-release-notes.md [hunk 6]
```diff
@@ -1,13 +1,59 @@
...
+
+## Swift SDK :
+
+- Added info clarifying "completed subscription state
+
+## Other:
+
+- Internal:
+  - Add shared backend source for the unit test suite suites
```
Comment (cbullinger) [inline docs-release-notes.md:37]: `suite suites` intentional?

File: docs-release-notes.md [hunk 7]
```diff
@@ -1,13 +1,59 @@
...
+  - C++ SDK:
+    - Change capitalization convention for constant variables in authentication unit tests
+  - Swift SDK:
+    - Use Swiftlint to fix linting errors
+  - .NET SDK:
+    - Replace "Task" model with "Item" model in unit test suite.
```
Comment (cbullinger) [inline docs-release-notes.md:43]:
```suggestion
    - Replace "Task" model with "Item" model in unit test suite
```

**Review by cbullinger — APPROVED** [review]
A couple small corrections

File: docs-release-notes.md (hunk 6 above)
Comment (mongomoe) [reply docs-release-notes.md:37]: Nope, should just be "unit test suites". Fixed now 👍

**Pull request merged**